scripts/spark_wdl/ReadsPipelineSpark.wdl

```diff
@@ -98,7 +101,8 @@ task ReadsPipelineSpark {
       -O ${output_vcf} \
       --known-sites ${known_sites} \
       -pairHMM AVX_LOGLESS_CACHING \
-      --maxReadsPerAlignmentStart 10 \
+      --max-reads-per-alignment-start 10 \
```

Comment:
Why do we use a lower downsampling rule than the regular HaplotypeCaller here? I thought it was default 30 for the non-spark version.

Reply:
I've removed this so it uses the default.